### DISCUSSION TITLE
Forums section disappeared

### DISCUSSION THREAD

**Discussion by marqpdx** [djangopackages/djangopackages#650] [Grids]
HI,

was linked to https://djangopackages.org/grids/g/forums/

and it returned 404, and said to post issue.

thx,
m

**Comment by rafalp** [comment]
I've brought it back. ;)

Forums section suffers from people not understanding how site works, and treating it like "forum" for asking questions. Its not first time I see somebody edit grid's name/description for something else. This time grid was renamed to "errors???" and its description was replaced with python question:

<img width="1011" alt="zrzut ekranu 2018-06-01 o 06 20 15" src="https://user-images.githubusercontent.com/750553/40820972-1380631e-6564-11e8-9658-36de267c134f.png">

**Comment by rafalp** [comment]
Perhaps grid rename or larger description edit should require admin/mod review and acceptance?

**Comment by marqpdx** [comment]
maybe it’s worth renaming as “Discussion Forums”; might reduce the confusion a tad.

thanks for restoring!
:)
m

**Comment by rafalp** [comment]
I don't think that renaming "Forums" to "Discussion Forums" would prevent people that obviously just follow "forums" keyword and then find first text area on the page that they can use to leave message. from vandalising this page further.

**Comment by ryancheley** [comment]
There is logic in the `base.html` that will use the url of the slug which should prevent this type of breaking from occurring in the future
